indico/modules/events/registration/client/js/form/fields/MultiChoiceInput.jsx

```diff
@@ -123,7 +124,7 @@ function MultiChoiceInputComponent({
               {withExtraSlots && (
                 <td>
                   {choice.isEnabled && (
-                    <Dropdown
+                    <Select
```

Comment:
It works fine in general, except for a weird issue (which does not happen in the SingleChoiceInput btw) that the first time one of the Select dropdowns is opened, there's a short flash of the dropdown having the full screen's width and positioned far below.

Reply:
Can you do a screen capture or give me some steps to reproduce? I'm not seeing anything weird here on either Firefox or Chrome.

Reply:
I wasn't able to repro this, but I identified some code that *might* be causing it, so I fixed it and updated this branch. Let me know if it changed anything.

Reply:
I think you forgot to push it ;)

Reply:
Sorry. Try now.

Reply:
yep, that fixed it, thanks!